package.json

```diff
@@ -21,20 +18,21 @@
     "nod-validate": "^2.0.12",
     "pace": "hubspot/pace#a03f1f1de62c9cea6c88b2267b8d7a83858b6cb6",
     "slick-carousel": "^1.8.1",
+    "svg-injector": "^1.1.3",
     "sweetalert2": "^6.10.1"
   },
   "devDependencies": {
     "@bigcommerce/citadel": "^2.15.1",
     "babel-core": "^6.26.0",
-    "babel-eslint": "^8.0.1",
-    "babel-loader": "^7.1.2",
-    "babel-plugin-dynamic-import-webpack": "^1.0.1",
-    "babel-plugin-lodash": "^3.2.11",
+    "babel-eslint": "^8.2.2",
+    "babel-loader": "^7.1.4",
+    "babel-plugin-dynamic-import-webpack": "^1.0.2",
+    "babel-plugin-lodash": "^3.3.2",
     "babel-plugin-syntax-dynamic-import": "^6.18.0",
     "babel-plugin-transform-regenerator": "^6.26.0",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.2.2",
-    "clean-webpack-plugin": "^0.1.17",
+    "babel-preset-env": "^1.6.1",
```

Comment:
Because we are using [useBuiltIns](https://github.com/bigcommerce/cornerstone/blob/master/webpack.common.js#L30), this will tree-shake babel-polyfill to just what we are using. However, I will also add a .browserslistrc file to set targets just in case.